test/test.js

```diff
@@ -86,6 +86,19 @@ test('passes headers', (t) => {
   })
 })
 
+test('request has rawHeaders', (t) => {
+  t.plan(2)
+  const dispatch = function (req, res) {
+    res.writeHead(200, { 'Content-Type': 'text/plain' })
+    res.end(req.rawHeaders.toString())
+  }
+
+  inject(dispatch, { method: 'GET', url: 'http://example.com:8080/hello', headers: { Super: 'duper' } }, (err, res) => {
+    t.error(err)
+    t.equal(res.payload, 'super,duper,user-agent,lightMyRequest,host,example.com:8080')
```

Comment:
Should not it be `Super` capital letter as the example here https://nodejs.org/api/http.html#http_message_rawheaders?

Reply:
We parse all the header to lowercase when the users apply.
https://github.com/fastify/light-my-request/blob/master/lib/request.js#L71

Reply:
> Maybe we can move the logic of inserting rawHeaders inside that loop.

not really because some headers (like user-agent, host) are set later:
https://github.com/fastify/light-my-request/blob/master/lib/request.js#L74-L75
https://github.com/fastify/light-my-request/blob/master/lib/request.js#L83
https://github.com/fastify/light-my-request/blob/master/lib/request.js#L102
https://github.com/fastify/light-my-request/blob/master/lib/request.js#L107